src/components/tooltip/index.tsx

```diff
@@ -50,38 +40,22 @@ interface Props {
 const Tooltip: FC<Props> = ({
   children,
   renderContent,
-  getPosition,
-  alignment = TooltipAlignment.center,
+  preferredPosition,
+  alignment = TooltipAlignment.middle,
 }) => {
-  const tooltipContainer = useRef(null)
-  const [isVisible, setIsVisible] = useState(false)
-  const [calculatedPosition, setCalculatedPosition] = useState(null)
-
   return (
-    <div
-      ref={tooltipContainer}
-      onClick={(event) => event.stopPropagation()}
-      onMouseEnter={(event) => {
-        event.stopPropagation()
-        setCalculatedPosition(
-          getPosition(
-            getSpaceAroundRect(tooltipContainer.current.getBoundingClientRect())
-          )
-        )
-        setIsVisible(true)
-      }}
-      onMouseLeave={(event) => {
-        event.stopPropagation()
-        setIsVisible(false)
-      }}
-      className="relative inline-block"
-    >
-      {isVisible ? (
-        <PopUp position={calculatedPosition} alignment={alignment}>
-          {renderContent()}
-        </PopUp>
-      ) : null}
-      {children}
+    <div className="inline-block" onClick={(e) => e.preventDefault()}>
```

Comment:
will links within the tooltip work with this?

Reply:
Yes, it's to enable having tooltips inside links (card)